AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -360,17 +360,23 @@ - (BOOL)removeAccount {
   return YES;
 }
 
-- (void)acquireTokenSilentlyWithMSALAccount:(MSALAccount *)account {
+- (void)acquireTokenSilentlyWithMSALAccount:(MSALAccount *)account withUIFallback:(BOOL)withUIFallback {
```

Comment:
`uiFallback:(BOOL)uiFallback`?

Reply:
I'd rather go for `withUIFallback:(BOOL)uiFallback` since its not a public API